packages/gluestick/transpile.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+transpile () {
+  local dirname=$1
+  local tmpdirname=$dirname"-tmp"
+
+  $(npm bin)/babel $dirname --presets=react,es2015,stage-0 --plugins=transform-decorators-legacy,transform-flow-strip-types -d $tmpdirname && rm -rf $dirname && mv $tmpdirname $dirname & \
```

Comment:
Why do we want to transpile in place like this?  This script can only run once before needing to revert the working directory.